components/input-group/src/overrides/select.js

```diff
@@ -1,13 +1,14 @@
 /** @jsx jsx */
 
-import { jsx } from '@westpac/core';
+import { jsx, getLabel } from '@westpac/core';
 import { Select as SelectInput } from '@westpac/text-input';
 
 const Select = ({ state: { size, data }, ...rest }) => (
 	<SelectInput size={size} data={data} {...rest} />
 );
 
-const selectStyles = (_, { position }) => ({
+const selectStyles = (_, { position, size }) => ({
```

Comment:
This was interesting!
I had to include `size` even though it was not used in the styles because it's used in `SelectInput` in the component and this label overrides the label from the `SelectInput`.

Reply:
^ should we make note of that in the code with a comment?